### ISSUE TITLE
What am i doing wrong?

### ISSUE THREAD

**Issue by Alejinho** [workshopper/stream-adventure#166]
I have the same proposed solution but...

```
module.exports = function(cmd,args){
        var cm = spawn(cmd,args);
        return duplexer(cm.stdin,cm.stdout);
};
```

`operator: equal
    expected:
      'volliq\nvcmwocn\nvinnfy\nviifyuh\nvuffiih\nviifyuh\nvivvch\nvir\nvyyn\nvun\nvsny\nvyynfy\nvomn'
    actual:
      'volliq\nvcmwocn'
    at: ConcatStream.<anonymous> (/usr/local/lib/node_modules/stream-adventure/node_modules/concat-stream/index.js:36:43)
`

**Comment by mcpengelly** [comment]
That is the correct solution, according to the lesson anyway. I had the same issue, I think that lesson is broken.

**Comment by ralphtheninja** [comment]
> That is the correct solution, according to the lesson anyway. I had the same issue, I think that lesson is broken.

I have the same problem

**Comment by kalinchernev** [comment]
@Alejinho could you by any chance test if the fix from #172 is solving it for you now?
Myself, ran your solution and it worked. Please test.

**Comment by ccarruitero** [comment]
fixed by #172
